src/AuditableTraitObserver.php

```diff
@@ -43,4 +44,18 @@ public function updating(Model $model)
             $model->updated_by = $this->getAuthenticatedUserId();
         }
     }
+
+    /**
+     * Model's deleting event hook
+     *
+     * @param Model $model
+     */
+    public function deleting(Model $model)
+    {
+        if (Schema::hasColumn($model->getTable(), 'deleted_by')) {
```

Comment:
Good solution. Maybe we should use `getDeletedByColumn()` function here.

Reply:
I had been just following the style of the code you'd written above, I've created a separate PR #5 to make the columns all configurable from model constants, just like how Laravel [changes the timestamp column names](https://laravel.com/docs/5.8/eloquent#eloquent-model-conventions).

Though this doesn't change the migration methods, think we'd need a config file to address that.  I think users would be expected to just create each column manually without the macro if they were to deviate from conventions anyway, as they would if they used non standard `created_at` and `updated_at` column names.